### DISCUSSION TITLE
Are the names of the node creators the same according to consensus.

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#425] [Q&A] [answered]
Hello 

A node creator is called a builder, or a miner with a PoW consensus.
And a validator with a PoS consensus ? 

What do you call the user who hides behind the node of a blockchain?

**Comment by cromewar** [comment, marked as answer]
Based on the official ethereum [docs](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/) they are just called "validators".

Best regards, Cromewar.

**Reply by wilonweb** [reply]
Thanks You, I discovery this world with FreeCodeCamp and you are Awesome !!! 
apart from the forum you work in the web or crypto ?

**Reply by cromewar** [reply]
I'm just a learner as you, actually participating on chainlink hackathon, but I still consider myself a student.
